backend/src/main/java/aiversum/backend/rest/ImageController.java

```diff
@@ -24,15 +27,22 @@ public ImageController(PropertiesConfig propertiesConfig) {
 
 
     @PostMapping(value = "/openai/{model}", produces = "application/json")
-    public String generateOpenaiImage(@PathVariable String model, @RequestBody ImageQuery imageQuery)  {
-
+    public String generateOpenaiImage(@PathVariable String model, @RequestBody ChatCommand chatCommand)  {
+
+        Optional<Map<String, String>> options = Optional.ofNullable(chatCommand.options());
+        String imageSize = options
```

Comment:
why we use map here?

Reply:
It's very generic structure. If we will add more properties, then those will be reflected in this map automatically. Map is also a very flexible format, because this class wants only few option parameters from all of them (I know that for now there are only two, but in the future will be more)